framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/KieServerDeploymentImpl.java

```diff
@@ -83,6 +92,41 @@ public String getSecureServiceName() {
         super.waitForScale();
         if (getInstances().size() > 0) {
             RouterUtil.waitForRouter(getUrl());
+            waitForSecureRouter();
```

Comment:
Just looking at the name of this method and the statement above it: shouldn't we move this "wait for secure url" to RouterUtil as well?

Reply:
this method shouldn't be moved to RouterUtil because we connecting to the KIE Server